src/components/chip/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FC, PropsWithChildren } from 'react';
+import {
+  Box,
+  Flex,
+  FlexProps,
+  RecipeVariantProps,
+  Text,
+  useRecipe,
+} from '@chakra-ui/react';
+
+import { chipRecipe } from 'src/themes/shared/recipes/chip';
+import { CheckmarkIcon } from 'src/components/icons/CheckmarkIcon';
+
+export type ChipProps = {
+  variant?: RecipeVariantProps<typeof chipRecipe>['variant'];
+  onClick?: () => void;
+  href?: string;
+  'aria-label'?: string;
+};
```

Comment:
```suggestion
export type ChipProps = {
  onClick?: () => void;
  href?: string;
  'aria-label'?: string;
} & RecipeVariantProps<typeof chipRecipe>;
```

This way:
- you don't have to match the property name
- you don't need to adjust the props if you rename `variant` to `selected` (or introduce a new styling variant like `disabled`)